images/capi/packer/config/containerd.json

```diff
@@ -2,5 +2,6 @@
     "containerd_version": "1.3.4",
     "containerd_sha256": "4616971c3ad21c24f2f2320fa1c085577a91032a068dd56a41c7c4b71a458087",
     "containerd_pause_image": "k8s.gcr.io/pause:3.2",
-    "containerd_additional_settings": null
+    "containerd_additional_settings": null,
+    "containerd_cri_socket": "/var/run/containerd/containerd.sock"
```

Comment:
isn't this the `/run/docker/...` socket?

Reply:
No, it needs to be `/var/run/containerd/containerd.sock`, because `packer/config/containerd.json` is a place for default config values, mainly for non-Flatcar distros.
On the other hand, `packer/qemu/qemu-flatcar.json` and `images/capi/packer/ami/flatcar.json` need to have configs for Flatcar, so `/run/docker/libcontainerd/docker-containerd.sock`.